### PR TITLE
README: Add documentation on difference with 'terraform graph'

### PR DESCRIPTION
**Pull request by xescugc**
So it' can be clearly seen the output of both of them

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +69,34 @@ or HCL module
 $ inframap generate --hcl ./my-module/ | graph-easy
 ```
 
+## Which is the difference with `terraform grah`
```
Comment (tormath1) [inline README.md:72]:
```suggestion
## What is the difference with `terraform grah`
```
?

File: README.md [hunk 2]
```diff
@@ -71,6 +69,34 @@ or HCL module
...
+
+[Terraform Graph](https://www.terraform.io/docs/commands/graph.html) outputs a dependency graph of all the resources. We try to go one step forward,
+if the Provider is one of the ones we support we apply a set of rules to make the output more useful by showing the most important information.
```
Comment (tormath1) [inline README.md:75]: > If the provider is supported by InfraMap, we apply a set of rules in order to show only relevant information. 

?

File: README.md [hunk 3]
```diff
@@ -71,6 +69,34 @@ or HCL module
...
+
+If the Provider is one of the ones we do now support, the output is much more close to the Terraform one, but we focus on resources so the Provider/Vars
```
Comment (tormath1) [inline README.md:77]: On the other hand, if the provider is not supported, the output will be closer to the Terraform one (without displaying provider / variable nodes)

**Comment by tormath1** [comment]
LGTM, RS :)

File: README.md [hunk 4]
```diff
@@ -71,6 +69,33 @@ or HCL module
 $ inframap generate --hcl ./my-module/ | graph-easy
 ```
 
+## What is the difference with `terraform grah`
+
+[Terraform Graph](https://www.terraform.io/docs/commands/graph.html) outputs a dependency graph of all the resources on the tfstate/HCL. We try to go one step forward,
+by trying to make it human readable
+
+On the other hand, if the provider is not supported, the output will be closer to the Terraform one (without displaying provider / variable nodes)
```
Comment (xlr-8) [inline README.md:77]: I'd simply say: _"If the cloud provider is not [...]"_

**Pull request merged**